### ISSUE TITLE
Get error when use Form!

### ISSUE THREAD

**Issue by thearabbit** [nicolaslopezj/meteor-react-form#11]
Have any pre install or not such as `materail ui package`?
I get error

``` js
=> Meteor server restarted                                              
W20160227-15:04:25.718(7)? (STDERR) Error when doing SSR. path:/post-cre
ate: Cannot read property 'Form' of undefined                           
W20160227-15:04:26.055(7)? (STDERR) TypeError: Cannot read property 'For
m' of undefined                                                         
W20160227-15:04:26.055(7)? (STDERR)     at new PostCreate (app/component
s/post_create.jsx:7:6)                                                  
```

**Comment by macrozone** [comment]
see https://github.com/nicolaslopezj/meteor-react-form/issues/5

**Comment by thearabbit** [comment]
Thanks, waiting ...
+1

**Comment by macrozone** [comment]
Maybe disable SSR until the problem is fixed. :+1:

**Comment by thearabbit** [comment]
Thanks, Excuse me could you example to use custom field?

``` js
<MRF.Form..............>
  <TextInput ...............>
  .................
</MRF.Form>
```

**Comment by nicolaslopezj** [comment]
A lot of fixes will come with the Meteor 1.3 version!
